index.html

```diff
@@ -11,33 +11,41 @@
     <link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet">
   </head>
   <body>
-    <header class="header">Etch a sketch</header>
+    <h1 class="header">Etch a Sketch</h1>
     <div class="js-radio-buttons section-button">
-      <form>
+      <label for="black">
```

Comment:
No need to provide `for` attribute if you wrap an input with a label. `for` is used when you want to attach a label to an input located on the page outside of `label` element.